tests/conftest.py

```diff
@@ -1,5 +1,34 @@
+from __future__ import annotations
+
 import os
+import socket
+import time
+import pytest
 
 
 def pytest_sessionstart(session):
-    os.environ["BM25"] = "bm25"
+	os.environ["BM25"] = "bm25"
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
The PR uses a pytest fixture in the `_postgres_healthcheck` but does not return any instance in the function. This is bad because the `pytest` fixture should return the instance for reusability during testing. I recommend deleting the `pytest` fixture decorator because this function should exist only for the health check of PostgreSQL.